orion/sites/admin/src/announce/announce.js

```diff
@@ -5,9 +5,62 @@ import ReactDOM from 'react-dom';
 
 export class AnnouncePage extends React.Component {
 	render() {
+		var numAnnouncements = 3; 	/* this needs to be a variable later */
+
+		/* these fake announcements will eventually be replaced with real announcements */
+		let fakeAnnounceA = {
+			date: "02/15/2020",
+			author: "Melon King",
+			message: "Only evil people waste watermelon"
+		};
+		let fakeAnnounceB = {
+			date: "02/16/2020",
+			author: "Fruit Hater",
+			message: "Watermelons are fruit, therefore they deserve to dry on the vine"
+		};
+		/* will use this to test text wrapping later */
+		let fakeAnnounceC = {
+			date: "02/23/2020",
+			author: "Talkative",
+			message: "This will be a longer message later"
+		};
 		return (
       <div id="view-announce">
-        <h1>All Announcements</h1>
+        <h1>All Announcements ({numAnnouncements}) </h1>
+				<ul> {/* this is the subheading */}
+					<div className="announce-lists">
+						<li className="li-med"> Date </li>
+						<li className="li-med"> Author </li>
+						<li className="li-large"> Message </li>
+						<li className="li-small"> </li>
+					</div>
+				</ul>
+
+				{/* these are the lists under the subheading */}
+				<ul>
+					<li className="announce-lists">
```

Comment:
I think maybe a better name could be "announce-list-item" or "announce-list-row"?
"announce-lists" implies multiple items or an entire list, when really you're only stying a single row.

Reply:
"announce-lists" applies to each announcement, so each list item under the subheading. Should I still change it?